scripts/sync-configs.sh

```diff
@@ -0,0 +1,15 @@
+#!/bin/bash
+
+echo "first make sure metrictank-sample.ini is up to date. its values should match the defaults used by metrictank. and comments should match the descriptions provided by metrictank help menus"
+echo "now we will run vimdiff to manually synchronize updates from sample config to other configs:"
+echo "try to make every config as closely resembling the sample config as possible, while retaining the customisations that makes each config unique"
+echo "hit a key when you're ready...."
+read
+for cfg in $(find . -name 'metrictank*.ini' | grep -v '\./metrictank-sample.ini'); do
+	vimdiff metrictank-sample.ini $cfg
+done
+
+echo "updating docs/config.md"
+./scripts/config-to-doc.sh > docs/config.md
```

Comment:
This assumes that you are running the script from $GOPATH/src/github.com/raintank/metrictank, which wont always be true.  What if a user is in scripts/ and runs ./sync-configs.sh?

We handle this in all other scripts with
```
# Find the directory we exist within
DIR=$( cd "$( dirname "${BASH_SOURCE[0]}" )" && pwd )
cd ${DIR}
```